ciao-cli/instance.go

```diff
@@ -519,9 +519,14 @@ func (cmd *instanceAddCommand) run(args []string) error {
 		fatalf(err.Error())
 	}
 
+	if len(servers.Servers) < cmd.instances {
```

Comment:
So if at least one instance starts, ciao-cli does not return an error?  Is this correct?  

If so, I think we should also change,  [bat.parseInstances](https://github.com/01org/ciao/blob/master/bat/compute.go#L519).  Currently, this function would return an error as it would not be able to parse "Some instances failed to start - check the event log for details."  What we really want it to do is to return information about the instances that were successfully created.  This will allow the calling BAT test to clean them up properly.  The other option would be to move the

"Some instances failed to start - check the event log for details."

after the server ids have been output but this is arguable more fragile.